packages/infra/config/hl7-notification-config.ts

```diff
@@ -11,5 +11,6 @@ export interface Hl7NotificationConfig {
 export type Hl7NotificationVpnConfig = {
   partnerName: string;
   partnerGatewayPublicIp: string;
+  partnerInternalCidrBlock: string;
```

Comment:
Are we planning to have a network per partner?

Reply:
Oh, this is the CIDR of their VPN on our infra, right?

Reply:
Yeah we will have one VPN per partner. I'm calling them partner because this same infra could be used to integrate with other orgs that are not HIEs.